stdlib/contrib/bonitoo-io/alerta/alerta.flux

```diff
@@ -0,0 +1,94 @@
+package alerta
+
+import "http"
+import "json"
+import "strings"
+
+// alert sends an alert to Alerta.
+// `url` - string - Alerta URL.
+// `apiKey` - string - Alerta API key.
+// `resource` - string - resource under alarm.
+// `event` - string - event name.
+// `environment` - string - environment. Valid values: "Production", "Development" or empty string (default).
+// `severity` - string - event severity. See https://docs.alerta.io/en/latest/api/alert.html#alert-severities.
+// `service` - arrays of string - list of affected services.
+// `group` - string - event group.
+// `value` - string - event value.
+// `text` - string - text description.
+// `type` - string - event type.
+// `origin` - string - monitoring component.
+// `timestamp` - time - time alert was generated.
+// `timeout` - int - seconds before alert is considered stale.
+alert = (
+    url,
+    apiKey,
+    resource,
+    event,
+    environment="",
+    severity,
+    service=[],
+    group="",
+    value="",
+    text="",
+    tags=[],
+    attributes,
+    origin="InfluxDB",
+    type="",
+    timestamp=now()
+) => {
+
+  alert = {
+      resource: resource,
+      event: event,
+      environment: environment,
+      severity: severity,
+      service: service,
+      group: group,
+      value: value,
+      text: text,
+      tags: tags,
+      attributes: attributes,
+      origin: origin,
+      type: type,
+      createTime: strings.substring(v: string(v: timestamp), start: 0, end: 23) + "Z" // Alerta supports ISO 8601 date format YYYY-MM-DDThh:mm:ss.sssZ only
```

Comment:
Would this benefit from a new function in the date package? Maybe something like `date.format`? This is just a question to guide future development. Since the function doesn't exist, this isn't something that needs to be changed.

Reply:
`flux fmt` applied. Indeed, if there was a date formatting function, it would be useful here.